src/bin/pg_autoctl/keeper_config.c

```diff
@@ -799,6 +799,22 @@ keeper_config_accept_new(KeeperConfig *config, KeeperConfig *newConfig)
 			newConfig->postgresql_restart_failure_max_retries;
 	}
 
+	if (config->pgSetup.ssl.active != newConfig->pgSetup.ssl.active)
+	{
+		log_info("Reloading configuration: ssl is now %s; used to be %s",
+				 newConfig->pgSetup.ssl.active ? "active" : "disabled",
+				 config->pgSetup.ssl.active ? "active" : "disabled");
+	}
+
+	if (config->pgSetup.ssl.sslMode != newConfig->pgSetup.ssl.sslMode)
```

Comment:
For consistency I guess this check should also be done for the other ssl fields, i.e. `caFile` and `crlFile`.

Reply:
Done now.